cmd/nodecmd/create.go

```diff
@@ -27,6 +27,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	useAWS                        bool
+	useGCP                        bool
+	cmdLineRegion                 string
+	authorizeAccess               bool
```

Comment:
I don't think we need this one as user doesn't need to authorize access if uses command line.
what was the point of adding all this params to command line args  and not authorizing access ?

Reply:
there is no other value than true for command to run
we ask user in UI because of politeness

Reply:
I am not sure about how to fix this. If not adding a flag, CLI is always going to add interaction and ask the user for authorization. We ask the user due because that was previously asked to law people. To cover ourselves. The flag is for advanced users. cc @sukantoraymond